Source/WTF/wtf/Forward.h

```diff
@@ -133,9 +133,12 @@ struct IdentityExtractor;
 template<typename T> struct KeyValuePairKeyExtractor;
 template<typename KeyTraits, typename MappedTraits> struct KeyValuePairTraits;
 template<typename KeyTypeArg, typename ValueTypeArg> struct KeyValuePair;
-template<typename Key, typename Value, typename Extractor, typename HashFunctions, typename Traits, typename KeyTraits> class HashTable;
+enum class ShouldValidateKey : bool { Yes, No };
```

Comment:
This is reversed: `No` should come first, and then `Yes`. We want `No == 0 and Yes == 1`

Reply:
Fixed =)